### ISSUE TITLE
Missing mixins

### ISSUE THREAD

**Issue by sqlninja** [vikdiesel/admin-one-vue-bulma-dashboard#10]
Just purchased the html version and seem to be getting an error on missing mixins for the nav bar.

It seems that touch and desktop mixins are not present in the mixins file.

Am I missing something where these should be generated somewhere else?

**Comment by vikdiesel** [comment]
Hi @sqlninja,

Thank you for buying Admin One. Could you give steps to reproduce an error?

**Comment by sqlninja** [comment]
Sorry, it was late when I posted and I wasn't very thorough.
That was lazy on my part...

I am running a rails (6) site, and I migrated all the scss files into my `assets/stylesheets/dashboard` folder

I already had bulma and extensions installed via the following gems:
``` ruby
gem 'bulma-extensions-rails'
gem 'bulma-rails', '~> 0.8.0'
```

The primary issues I faced with getting the items to "function" were:
- some updates to the imports (removing `node_modules` from the import path since it's already included in the assets path)
- adding numerous variables that didn't seem to exist (for example: `$turquoise` or `$radius-rounded`)
- and the main issue originally stated, missing "breakpoint" mixins (i.e touch, desktop, until-widescreen, widescreen)
- lastly, I'm having an issue webpack, but fairly sure that is completely unrelated

So to summarize:
1. Downloaded assets from purchase
2. moved scss file into assets folder
3. updated imports
4. declared missing vars
5. missing breakpoint mixins

I suppose there could be a file that I missed that included them, but upon search the downloaded assets I did not see them.

Thanks for the quick reply!

**Comment by vikdiesel** [comment]
Hi @sqlninja, thank you. 

Breakpoint mixins are part of Bulma. I'm not sure how build process is done with rails, but it really looks like either some imports are missing, or the order of imports is wrong.

I would recommend to follow the logic of `src/scss/main.scss` which imports everything required in correct order. It could be you'll need `gem "bulma-rails", "~> 0.9.1"`, as project is built on Bulma 0.9.x.

**Comment by sqlninja** [comment]
Ok so I think I got this worked out...
Turns out that due to the bulma gems I need to import them into the theme scss files.
Also bumping the version fixed the missing breakpoint mixin issue.

Thanks for the pointers.
